index.html

```diff
@@ -52,5 +60,173 @@ <h1 class="secondHeroIntroHeading">
             </div>
         </div>
     </section>
+    <section>
+        <div class="workTextContainer">
+          <h2 class="workText">My Recent Works</h2>
+        </div>
+        <div class="indicator">
+          <div class="indicatorImg"></div>
+        </div>
+      </section>
+      <section class="workContainer">
+        <div class="firstContainer">
+          <div class="secondContainer">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <ul class="listLanguages">
+              <li class="languages">Ruby on rails</li>
+              <li class="languages">css</li>
+              <li class="languages">JavaScript</li>
+              <li class="languages">html</li>
+            </ul>
+            <div class="secondContainerBtn">
+              <button type="button" class="see-project click">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="firstContainer">
+          <div class="secondContainer">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <ul class="listLanguages">
+              <li class="languages">Ruby on rails</li>
+              <li class="languages">css</li>
+              <li class="languages">JavaScript</li>
+              <li class="languages">html</li>
+            </ul>
+            <div class="secondContainerBtn">
+              <button type="button" class="see-project click">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="firstContainer">
+          <div class="secondContainer">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <ul class="listLanguages">
+              <li class="languages">Ruby on rails</li>
+              <li class="languages">css</li>
+              <li class="languages">JavaScript</li>
+              <li class="languages">html</li>
+            </ul>
+            <div class="secondContainerBtn">
+              <button type="button" class="see-project click">See Project</button>
+            </div>
+          </div>
+        </div>
+        <div class="firstContainer">
+          <div class="secondContainer">
+            <h4>Multi-Post Stories Gain+Glory</h4>
+            <ul class="list-languages">
+              <li class="languages">Ruby on rails</li>
+              <li class="languages">css</li>
+              <li class="languages">JavaScript</li>
+              <li class="languages">html</li>
+            </ul>
+            <div class="secondContainerBtn">
+              <button type="button" class="see-project click">See Project</button>
```

Comment:
- Some of your works are not displaying as expected
![Screenshot from 2023-04-28 12-37-22](https://user-images.githubusercontent.com/97834160/235127722-91a418a3-8803-4d48-ad10-de9860c1a867.png)

This is Probably due to mispelling class names for some works. Please Fix this issue